festim/generic_simulation.py

```diff
@@ -343,7 +343,7 @@ def display_time(self):
         msg = "{:.1f} %        ".format(simulation_percentage)
         msg += "{:.1e} s".format(simulation_time)
         msg += "    Ellapsed time so far: {:.1f} s".format(elapsed_time)
-        if self.t != self.settings.final_time:
+        if self.t != self.settings.final_time and self.loglevel == 40:
```

Comment:
The tests are failing because there's a typo here:
```suggestion
        if self.t != self.settings.final_time and self.log_level == 40:
```